pkg/systemd/logs.html

```diff
@@ -67,7 +67,27 @@
 
         <div class="filter text-search">
             <label for="journal-grep" translate>Text</label>
-            <input id="journal-grep" class="form-control" type="text" translate="placeholder" placeholder="Type to filter">
+            <span class="filter-group">
+                <input id="journal-grep" class="form-control" type="text" translate="placeholder" placeholder="Type to filter">
+                <div id="logs-help" class="dropdown">
+                  <a role="link" tabindex="0" id="logs-help-toggle" class="dropdown-toggle dropdown-form-control" data-toggle="dropdown">
+                    <span class="pficon pficon-help"></span>
+                  </a>
+                  <div id="logs-help-menu" class="dropdown-menu">
+                    <span translate>The text entry allows searching the full content of the log entries. It accepts strings such as:</span>
+                    <ul>
+                        <li><span translate>qualifiers</span>, <span translate>E.g</span> 'priority', 'identifier', 'services'</li>
```

Comment:
s/E.g/e.g./. I also think this should mention the colon. Also, it seems it's "service:", not ":service*s*"